Annex60/Fluid/Movers/Data/SpeedControlled_Nrpm.mo

```diff
@@ -5,12 +5,21 @@ record SpeedControlled_Nrpm "Generic data record for FlowMachine_Nrpm"
   parameter Modelica.SIunits.Conversions.NonSIunits.AngularVelocity_rpm N_nominal = 1500
     "Nominal rotational speed for flow characteristic";
 
+  parameter Real[:] speeds(each final unit="1/min") = {0}
```

Comment:
- The units should be `Modelica.SIunits.Conversions.NonSIunits.AngularVelocity_rpm` and not `Real`.
- Because we used `N_nominal`, this should be `N` as it is the physical quantity for which `N_nominal` is its nominal value, no?